haskell/private/cc_libraries.bzl

```diff
@@ -79,6 +80,7 @@ def get_library_files(hs, cc_libraries_info, libraries_to_link, dynamic = False,
       libraries_to_link: list of LibraryToLink.
       dynamic: Whether dynamic libraries are preferred.
       pic: Whether position independent code is required.
+      follow_symlinks: Whether to return the actual path of the libraries or their symbolic link in the _solib_k8 directory
```

Comment:
This sounds a bit misleading. IIUC this will return both the actual library file and the symbolic link, not either or.

Reply:
Indeed. I've renamed it to `include_real_paths`. Not really pretty, but should be less misleading

Reply:
Thanks, I think that clarifies it well.